plugins/package-managers/python/src/main/kotlin/Poetry.kt

```diff
@@ -115,4 +118,14 @@ class Poetry(
     }
 }
 
-private val SCOPES_NAMES = listOf("main", "dev")
+private fun parseScopeNamesFromPyProject(pyprojectFile: File): List<String> {
```

Comment:
In [1] the `pyproject-toml` seems to specify dependencies a bit differently.
So, it should handle e.g. `[tool.poetry.group.dev.dependencies]`?

Note: 
- `.group`
- suffix: `.dependencies` instead of `-dependencies`



[1] 

[tool.poetry.group.dev.dependencies]